### PR TITLE
Add 2 remaining docs pages

### PR DESCRIPTION
**Pull request by katy-sadowski**
Addresses #542

### CODE REVIEW

File: vignettes/checks/plausibleGenderUseDescendants.Rmd [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "plausibleGender"
-author: ""
+title: "plausibleGenderUseDescendants"
```
Comment (katy-sadowski) [inline vignettes/checks/plausibleGenderUseDescendants.Rmd:2]: this needs review

File: vignettes/checks/plausibleUnitConceptIds.Rmd [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "plausibleUnitConceptIds"
-author: ""
+author: "Katy Sadowski"
```
Comment (katy-sadowski) [inline vignettes/checks/plausibleUnitConceptIds.Rmd:3]: this needs review

File: vignettes/checks/plausibleGenderUseDescendants.Rmd (hunk 1 above)
Comment (samhillman) [reply vignettes/checks/plausibleGenderUseDescendants.Rmd:2]: Reviewed, agree with content

File: vignettes/checks/plausibleUnitConceptIds.Rmd (hunk 2 above)
Comment (samhillman) [reply vignettes/checks/plausibleUnitConceptIds.Rmd:3]: Reviewed, agree with content

**Review by samhillman — APPROVED** [review]
Approve

**Review by samhillman — APPROVED** [review]

**Pull request merged**